plugins/emoticons.md

```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoticons without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open an inline dialog showing matching emoticons.
```

Comment:
`inline dialog` is the wrong term here, it should be an `inline toolbar`, `popup toolbar` or something like that instead. I'm honestly not sure what the term is for this, as we've never really defined it.

Reply:
updated per our discussion to: emoticon picker